src/collectors/diskspace/diskspace.py

```diff
@@ -146,7 +146,8 @@ def get_file_systems(self):
                      mount_point.startswith('/sys'))):
                     continue
 
-                if '/' in device and mount_point.startswith('/'):
+                if ((('/' in device or device == 'tmpfs') and
```

Comment:
the condition "'/' in device" filters it out


On Thu, May 5, 2016 at 10:30 AM, Grant Ridder <notifications@github.com>
wrote:

> In src/collectors/diskspace/diskspace.py
> <https://github.com/python-diamond/Diamond/pull/471#discussion_r62222126>:
>
> > @@ -146,7 +146,8 @@ def get_file_systems(self):
> >                       mount_point.startswith('/sys'))):
> >                      continue
> >
> > -                if '/' in device and mount_point.startswith('/'):
> > +                if ((('/' in device or device == 'tmpfs') and
>
> Maybe it would be better to add tmpfs to the filesystems list instead?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/python-diamond/Diamond/pull/471/files/17d872376ee741c1f49a6e67c06a9a5cd7f2138d#r62222126>
>